pkgs/applications/misc/ding/default.nix

```diff
@@ -0,0 +1,52 @@
+{
+aspell,
+gnugrep,
```

Comment:
The tool uses agrep and egrep internally. Is it correct to specify runtime dependencies like this?